corona-warn-app/cwa-app-android
# Screen: Vaccination List (EXPOSUREAPP-6734)

### Screen Vaccination List (with 2 states - Incomplete and Complete Vaccination)
- the displayed data is hardcoded
- once the real repo is available, we will show the data from there

How to test: 
- Test Menu => Vaccination => Vaccination List
- change `val vaccinationStatus = INCOMPLETE` to `val vaccinationStatus = COMPLETE` in `VaccinationListViewModel` to see the complete state
- Check on different devices, dark mode, switch font sizes, accessibility, etc. 

### Incomplete Vaccination State
![image](https://user-images.githubusercontent.com/10398034/117127644-a0d8d380-ad9c-11eb-914e-60370b4422cf.png)

### Complete Vaccination State
![image](https://user-images.githubusercontent.com/10398034/117148285-63813f80-adb6-11eb-936e-78f3babaca44.png)
